packages/docs/docs/charting/capturing-vital-signs.md

```diff
@@ -0,0 +1,216 @@
+| `value[x] `        | The actual result(s) of the observation.                                                                                                                                                                                                                     | [See below](#observation-datatypes)                                                                 |                                                  |
+| `dataAbsentReason` | The reason why the result is missing.                                                                                                                                                                                                                        | [Observation data absent reason types](http://hl7.org/fhir/R4/valueset-data-absent-reason.html)     | Asked But Unknown                                |
+| `interpretation`   | A categorical assessment of the result. For example, high, low, or normal.                                                                                                                                                                                   | [Observation interpretation types](http://hl7.org/fhir/R4/valueset-observation-interpretation.html) | Normal                                           |
+| `device`           | The device used to generate the measurement. This can be medical or non-medical and can range from a simple tongue depressor to an MRI machine. Also includes personal and wearable devices such as smart phones or watches                                  |                                                                                                     | Device/my-apple-watch                            |
+| `specimen`         | The specimen from which the observation was derived. A sample of material taken from the patient on which laboratory testing or analysis was performed.                                                                                                      |                                                                                                     | Specimen/finger-prick-blood-sample               |
+| `component`        | Some observations have multiple component observations. These component observations are expressed as separate code value pairs that share the same attributes. An example is systolic and diastolic component observations for blood pressure measurements. | [See below](#multi-component-observations)                                                          |                                                  |
+
+To record the actual _value_ of a measurement, you can use one of the various `value` fields on the `Observation` resource. There are fields for `valueQuantity`, `valueString`, `valueBoolean` and more.
+
+<details><summary>Examples</summary>
```

Comment:
These examples are great, but would like to see them split out as separate `<details>` elements
Such as: 
`<details><summary>Example: basic reading of body temperature</summary> ... </details>`

Also, using `js` instead of `json` for the code block type - it's more forgiving for things like comments